zio-http/src/main/scala/zhttp/http/headers/HeaderGetters.scala

```diff
@@ -278,7 +278,12 @@ trait HeaderGetters[+A] { self =>
 
   final def getSetCookiesDecoded: List[Cookie] =
     getHeaderValues(HeaderNames.setCookie)
-      .map(Cookie.decodeResponseCookie)
+      .map(Cookie.decodeResponseCookie(_))
+      .collect { case Some(cookie) => cookie }
+
+  final def getSignedSetCookiesDecoded(secret: String): List[Cookie] =
```

Comment:
we don't need this method. We can use `getSetCookiesDecoded(secret: Option[String])` instead.